docs/Quick-start-and-introduction.md

```diff
@@ -106,7 +106,16 @@ $this->loadComponent('Authentication.Authentication', [
 ]);
 ```
 
-## Accessing the user / identity data
+Once loaded, the `AuthenticationComponent` will require that all actions have an
+authenticated user present, but perform no other access control checks. You can
+disable this check for specific actions using `allowUnauthorized()`:
+
+```php
+// In your controller's beforeFilter method.
+$this->Authentication->allowUnauthorized(['view']);
```

Comment:
allowUnauthorized() inside the Authentication module might a sth to trip over?
Should we call it `allowUnauthenticated()` or just `allowPublic()`?

Reply:
The docs were wrong, and fixed in 1f325dc. The method is `allowUnauthenticated()`. I was worried that `allowPublic()` could be confused with PHP method visibility.